yaml/com/ibm/License/Entry/LicenseEntry.interface.yaml

```diff
@@ -0,0 +1,67 @@
+description: >
+    Implement to represent a license which enables features on system
+    Customer installs license to unlock certain resources or features
+    Each object implements xyz.openbmc_project.License.Entry interface
+    which has Name and Status properties
+    and implements xyz.openbmc_project.Object.Delete interface.
+properties:
+
+    - name: Name
+      type: string
+      description: >
+         Name of license
+    - name: SerialNum
+      type: string
+      description: >
+         The serial number of the license
+    - name: Type
+      type: enum[self.Type]
+      default: Trial
+      description: >
+          The current type of the license
+    - name: AuthorizationType
+      type: enum[self.AuthorizationType]
+      default: Unlimited
+      description: >
+          The current Authorization type of the license
+    - name: ExpirationTime
+      type: string
+      description: >
+         The date when the license expires
+    - name: AuthDeviceNumber
+      type: uint32
+      default: 0
+      description: >
+         The number of devices authorized by the license
+    - name: Duration
+      type: string
```

Comment:
We don't have a Duration type? 
Should we use something like int and number of seconds remaining? 
I don't think a string makes sense here

Reply:
Duration can take values as below
https://en.wikipedia.org/wiki/ISO_8601#Durations

.Duration                                             property  s         "PT720H"                                 emits-change writable

Thus its a string

Reply:
I really think we should use like an int here. 
If we really do want a string then lets document that format (but parsing strings and assuming formats should error prone.

Reply:
Redfish schema expects this as string
                "RemainingDuration": {
                    "description": "The remaining length of time before the license expires.",
                    "longDescription": "This property shall contain the remaining length of time before the license expires.",
                    "pattern": "-?P(\\d+D)?(T(\\d+H)?(\\d+M)?(\\d+(.\\d+)?S)?)?",
                    "readonly": true,
                    "type": "string"
                },

Reply:
Removed this attribute in the latest patchset